File: design-patterns/csharp/csharp-composite-design-pattern/HumanResources.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace csharp_composite_design_pattern
{
    public class HumanResources : IEmployee
    {
        private List<IEmployee> employeeList;
        private string test;

        public HumanResources(string test){
            this.test = test;
            employeeList = new List<IEmployee>();
        }

        public void AddEmployee(IEmployee employee)
        {
            employeeList.Add(employee);
        }

        public void RemoveEmployee(IEmployee employee)
        {
            employeeList.Remove(employee);
        }

        public void ShowInfo()
        {
            Console.WriteLine(test);
            foreach (var employee in employeeList)
            {
                employee.ShowInfo();
            }
        }
    }
}

File: design-patterns/python/python-composite-design-pattern/HumanResources.py
from IEmployee import IEmployee

class HumanResources(IEmployee):
    def __init__(self, deneme):
        self.deneme = deneme  
        self.employee_list = [] 

    def add_employee(self, employee):
        self.employee_list.append(employee)

    def remove_employee(self, employee):
        self.employee_list.remove(employee)

    def show_info(self):
        print(self.deneme)
        for employee in self.employee_list:
            employee.show_info()